cos-fleetshard-operator/src/main/java/org/bf2/cos/fleetshard/operator/connector/ConnectorController.java

```diff
@@ -393,6 +403,50 @@ private UpdateControl<ManagedConnector> handleAugmentation(ManagedConnector conn
             .withName(connector.getSpec().getDeployment().getSecret())
             .get();
 
+        ConfigMap configMap = kubernetesClient.configMaps()
+            .inNamespace(connector.getMetadata().getNamespace())
+            .withName(ConfigMaps.generateConnectorConfigMapId(connector.getSpec().getDeploymentId()))
+            .get();
+
+        if (configMap == null) {
+            LOGGER.info(
+                "Configmap not found (cluster_id: {}, namespace_id: {}, connector_id: {}, deployment_id: {}), creating a new one",
+                connector.getSpec().getClusterId(),
+                connector.getMetadata().getNamespace(),
+                connector.getSpec().getConnectorId(),
+                connector.getSpec().getDeploymentId());
+
+            configMap = new ConfigMap();
+            configMap.setMetadata(new ObjectMeta());
+            configMap.getMetadata().setNamespace(connector.getMetadata().getNamespace());
+            configMap.getMetadata().setName(ConfigMaps.generateConnectorConfigMapId(connector.getSpec().getDeploymentId()));
+
+            Resources.setLabels(
+                configMap,
+                LABEL_CLUSTER_ID, connector.getSpec().getClusterId(),
+                LABEL_CONNECTOR_ID, connector.getSpec().getConnectorId(),
+                LABEL_DEPLOYMENT_ID, connector.getSpec().getDeploymentId());
+
+            Resources.setOwnerReferences(
+                configMap,
+                connector);
+
+            // copy operator type
+            Resources.setLabel(
+                secret,
+                LABEL_OPERATOR_TYPE,
+                connector.getMetadata().getLabels().get(LABEL_OPERATOR_TYPE));
+            try {
+                this.kubernetesClient.configMaps()
+                    .inNamespace(configMap.getMetadata().getNamespace())
+                    .withName(configMap.getMetadata().getName())
+                    .create(configMap);
+            } catch (Exception e) {
+                LOGGER.warn("", e);
```

Comment:
Why the use of log-and-rethrow here? Since we don't add anything to the message, can't we just not catch?

Reply:
I was just being consistent with https://github.com/bf2fc6cc711aee1a0c2a/cos-fleetshard/blob/e13e6f5bd656c778a6b718f8d09100f1f2b60c82/cos-fleetshard-sync/src/main/java/org/bf2/cos/fleetshard/sync/resources/ConnectorDeploymentProvisioner.java#L333-L336 but you have a point.

Reply:
I also don't know why we have this one :P

Reply:
AFAIK it was to avoid the exception to be swallowed, maybe it is not more the case but it was in the past

Reply:
done